receiver/hostmetricsreceiver/internal/scraper/processscraper/process.go

```diff
@@ -49,45 +49,25 @@ type commandMetadata struct {
 
 func (m *processMetadata) initializeResource(resource pdata.Resource) {
 	attr := resource.Attributes()
-	attr.InitEmptyWithCapacity(6)
-	m.insertPid(attr)
-	m.insertExecutable(attr)
-	m.insertCommand(attr)
-	m.insertUsername(attr)
-}
-
-func (m *processMetadata) insertPid(attr pdata.AttributeMap) {
+	attr.EnsureCapacity(6)
```

Comment:
This is a change in behavior. Previously it would erase any existing attributes. Now it preserves them. Is this desirable?

Reply:
Not a problem, the output param is empty.